common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -171,18 +169,14 @@ public void run() {
     @Override
     public boolean onBackPressed() {
         Logger.info(TAG, "Back button is pressed");
-        if (null != mWebView && mWebView.canGoBack()) {
-            // User should be able to click back button to cancel. Counting blank page as well.
-            final int BACK_PRESSED_STEPS = -2;
-            if (!mWebView.canGoBackOrForward(BACK_PRESSED_STEPS)) {
-                cancelAuthorization(true);
-            } else {
-                mWebView.goBack();
-            }
-            return true;
+
+        if (mWebView.canGoBack()) {
```

Comment:
We lost a null check here.  Did we need it?

Reply:
Nope, not sure why this was here to begin with. The view will be created as part of the Fragment's start-up lifecycle